### PR TITLE
Upgrade Minim to 0.14.0 and Release 2.0.0

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

**Comment by pksunkara** [comment]
Dependent on https://github.com/refractproject/minim-parse-result/pull/7

**Comment by smizell** [comment]
@pksunkara still failing after published changes. Looks like there is a test using compact Refract.

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.1.0 - 2016-04-28
+
+- Upgrade Minim to 0.14.0 and thus remove the short hand notation in fury
```
Comment (smizell) [inline Changelog.md:3]: Isn't this a breaking change?

Comment (pksunkara) [inline Changelog.md:3]: Should we release 2? I don't think anyone is using compact refract except in tests.

File: test/fury.js [hunk 2]
```diff
@@ -223,7 +349,7 @@ describe('Refract loader', () => {
       annotation = result.get(1);
     });
 
-    context('parse result annotation', () => {
+    context('and parse result annotation', () => {
```
Comment (smizell) [inline test/fury.js]: Nitpick: I think this should be something different, along with line 342. IMO, it should be:

``` js
// line 342
context('when given a valid API'. function() {
  // All `it` assertions here
});
```

The "and" feels off to me.

File: Changelog.md (hunk 1 above)
Comment (smizell) [inline Changelog.md:3]: I'm not sure, though if we are going by SemVer, this is a breaking change, isn't it?

Comment (pksunkara) [inline Changelog.md:3]: Yeah. I have no issues doing v2 now but we will be adding the compact refract back anyway, which is why I did v1.1. I will leave the decision upto you.

**Pull request merged**